src/config/config.cc

```diff
@@ -227,7 +221,7 @@ Config::Config() {
       {"migrate-batch-rate-limit-mb", false, new IntField(&migrate_batch_rate_limit_mb, 16, 0, INT_MAX)},
       {"unixsocket", true, new StringField(&unixsocket, "")},
       {"unixsocketperm", true, new OctalField(&unixsocketperm, 0777, 1, INT_MAX)},
-      {"log-retention-days", false, new IntField(&log_retention_days, -1, -1, INT_MAX)},
+      {"log-retention-days", true, new IntField(&log_retention_days, -1, -1, INT_MAX)},
```

Comment:
So, `spdlog` cannot change the retention days after initialization?

Reply:
Currently yes.